samcli/lib/telemetry/event.py

```diff
@@ -110,7 +113,10 @@ def track_event(event_name: str, event_value: str):
                 EventTracker.track_event("UsedFeature", "FeatureY")
                 return some_value
         """
-        EventTracker._events.append(Event(event_name, event_value))
+        try:
+            EventTracker._events.append(Event(event_name, event_value))
+        except EventCreationError as e:
```

Comment:
just one last question - is `EventCreationError` the only exception that can occur? Any possibility that `EventTracker._events.append` would fail?

Reply:
There's no situation I can see where `append` would fail, since it's a standard list operation. The only place I can see that may go wrong is something being thrown when the Event itself is created, which is why I capture that error specifically.

Reply:
sounds good. makes sense to me.